tests/config/format-test.js

```diff
@@ -52,7 +52,20 @@ const unstableTests = new Map(
   }),
 );
 
-const unstableAstTests = new Map();
+const unstableAstTests = new Map(
+  [
+    ["angular/icu-expression/basic-select-01.html", () => true],
+    ["angular/icu-expression/basic-select-02.html", () => true],
+    ["angular/icu-expression/basic-plural-01.html", () => true],
+    ["angular/icu-expression/basic-plural-02.html", () => true],
+    ["angular/icu-expression/basic-plural-03.html", () => true],
+    ["angular/icu-expression/basic-plural-04.html", () => true],
+    ["angular/icu-expression/combination-01.html", () => true],
```

Comment:
I couldn't figure out how to resolve the FULL_TEST failure caused by rewriting `file.content` of `ParseLocation`...

Reply:
You don't know how AST compare works? https://github.com/prettier/prettier/blob/e04f0b023bda237c84280d244ee39bb45c64a44e/src/language-html/clean.js#L23

Reply:
Thank you, updated [5d42eda](https://github.com/prettier/prettier/pull/15777/commits/5d42eda5d2b788e860b46724de7635a4ad87d33b)